skeleton/backstage/template.yaml

```
@@ -141,6 +141,7 @@ spec:
           description: sed.edit.CATALOG_DESCRIPTION 
           dockerfile: sed.edit.DOCKERFILE
           buildContext: sed.edit.BUILDCONTEXT
+          gitopsSecretName: ${{ 'gitops-auth-secret' if parameters.hostType === 'GitHub' else 'gitlab-gitops-auth-secret' }}
```

```suggestion
          gitopsSecretName: ${{ 'gitops-auth-secret' if parameters.hostType === 'GitHub' else 'gitlab-auth-secret' }}
```

since we are using `gitlab-auth-secret` in repo CR already, we will continue use that, but needs a username set for that secret. 
installer issue: https://issues.redhat.com/browse/RHTAPBUGS-1191